static/favicons/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
+    "short_name": "",
+    "icons": [
+        {
+            "src": "android-chrome-192x192.png",
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "android-chrome-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#4f62ad",
+    "background_color": "#4f62ad",
+    "display": "standalone"
+}
```

Comment:
Some mandatory fields seem to be missing, such as `display_override`.

Reply:
Was generated with https://realfavicongenerator.net/ 
I think it is not there because this attribute is not suitable for all browsers, a lot in the experiment